src/native/corehost/fxr/sdk_resolver.cpp

```diff
@@ -353,6 +420,36 @@ bool sdk_resolver::parse_global_file(pal::string_t global_file_path)
         }
     }
 
+    const auto& paths_value = sdk->value.FindMember(_X("paths"));
+    if (paths_value != sdk->value.MemberEnd() && !paths_value->value.IsNull())
+    {
+        if (!paths_value->value.IsArray())
+        {
+            trace::warning(_X("Expected an array for 'sdk/paths' value in [%s]"), global_file_path.c_str());
+            return false;
+        }
+
+        has_custom_paths = true;
+        const auto& paths_array = paths_value->value.GetArray();
+        paths.reserve(paths_array.Size());
+        for (const auto& path : paths_array)
+        {
+            paths.push_back(path.GetString());
```

Comment:
I assume that `GetString()` will throw if it isn't a string? If so, do we want to show a warning/log instead of a crash?

Reply:
Added the check and trace.